### ISSUE TITLE
Node-sass version outdated

### ISSUE THREAD

**Issue by EpicOkapi** [dlmanning/gulp-sass#621]
The current version of node-sass, used by gulp-sass, is outdated and doesn't always work properly with node v8.

**Comment by wolfitoXtreme** [comment]
Same here, first of all thanks for this awesome plugin;)
I'm struggling with errors resulting from bug with libsass, that is supposed to be already fixed there.
-> Force evaluation of results from map-get. Fixes #2309
Is is possible to update NodeSASS in order to get the fix?

**Comment by WraithKenny** [comment]
Just as a note: https://github.com/dlmanning/gulp-sass/wiki/Common-Issues-and-Their-Fixes#node-sass-version-not-up-to-date describes how to solve that issue without needing this repo to update!

**Comment by pd93** [comment]
Hey, this is currently breaking the issue fixed in https://github.com/sass/node-sass/issues/1589 for me. `node-sass` now provide binaries for Alpine, but the latest Alpine version is not supported by `gulp-sass` because the version of `node-sass` included is too low.

`npm install` logs:
```
Step 9/11 : RUN npm i
 ---> Running in 409b8fd74c75

> node-sass@4.5.3 install ...../node_modules/node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.5.3/linux_musl-x64-59_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.5.3/linux_musl-x64-59_binding.node":

HTTP error 404 Not Found
.....
```

Note the `linux_musl-x64-59_binding.node`. This binding is only available with `node-sass` v4.6.0+. `gulp-sass` currently uses v4.5.3.

It's also worth noting that the workaround by @WraithKenny will not fix the issue as the old v4.5.3 binary will still be built - which is very slow (compared to downloading a binary).

The workaround for this at the moment is to downgrade the Alpine version.

**Comment by isochronous** [comment]
node-sass is currently missing several windows binaries for 4.x releases, including the version installed by default with v3.1.0.  This is causing gulp-sass to not work without manually compiling node-sass, which is a hit-or-miss process on windows machines.
